Sources/Target/Windows/Thread.cpp

```diff
@@ -245,11 +245,26 @@ void Thread::updateState(DEBUG_EVENT const &de) {
     _stopInfo.reason = StopInfo::kReasonThreadExit;
     break;
 
-  case OUTPUT_DEBUG_STRING_EVENT:
+  case OUTPUT_DEBUG_STRING_EVENT: {
+    auto const &DSInfo = de.u.DebugString;
```

Comment:
Mhmm, I think you're right. I'll fix it before checking in if travis passes
(don't want to get pulled down the queue again).

On Thu, May 12, 2016 at 5:29 PM, Francis Ricci <notifications@github.com>
wrote:

> In Sources/Target/Windows/Thread.cpp
> <https://github.com/facebook/ds2/pull/344#discussion_r63118655>:
>
> > @@ -245,11 +245,26 @@ void Thread::updateState(DEBUG_EVENT const &de) {
> >      _stopInfo.reason = StopInfo::kReasonThreadExit;
> >      break;
> >
> > -  case OUTPUT_DEBUG_STRING_EVENT:
> > +  case OUTPUT_DEBUG_STRING_EVENT: {
> > +    auto const &DSInfo = de.u.DebugString;
>
> dsInfo is probably more consistent camel-case?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/344/files/faab52bb452d770e082f2574350e5fa9f51ef3d6#r63118655>
>



-- 
Stephane Sezer